Makefile

```diff
@@ -55,6 +55,10 @@ integration-test: build
 	@go test -tags=cleanup -v ./...
 	@go test -tags=integration -v ./...
 
+integration-test-v1: build
+	@go test -tags=cleanup -v ./...
+	@go test -tags=integration_v1 -v ./...
```

Comment:
Add `integration_v1` to `PHONY`
btw in case v1 and v2 tests are executed in parallel, cleanup would interfere there causing tests to fail. Should cleanup be the part of integration test?

Reply:
> Add integration_v1 to PHONY

True, forgot about adding the target to `PHONY` :+1: 

> btw in case v1 and v2 tests are executed in parallel, cleanup would interfere there causing tests to fail. Should cleanup be the part of integration test?

You are right, better would be to have a dedicated `cleanup` target. I'll take this with me for the follow-up PR.